lib/pagseguro/transaction/collection.rb

```diff
@@ -0,0 +1,20 @@
+module PagSeguro
+  class Transaction
+    class Collection
+      extend Forwardable
+
+      def_delegators :@transactions, :each, :empty?, :any?
+
+      def transactions=(objects)
```

Comment:
I think these are not `transactions`, but `statuses`, `transaction_statuses` or something like that